Jenkinsfile

```diff
@@ -104,7 +104,8 @@ pipeline {
         withGithubNotify(context: 'Lint') {
           deleteDir()
           unstash 'source'
-          golang(){
+          setEnvVar('GO_VERSION', readFile(file: "${BASE_DIR}/.go-version").trim())
```

Comment:
Is this still required in this stage? I see the env var already defined for another one here: https://github.com/elastic/apm-server/pull/4456/commits/8d090edcf7ef07a8af342b9fb669d8cfc0781b43#diff-e6ffa5dc854b843b3ee3c3c28f8eae2f436c2df2b1ca299cca1fa5982e390cf8L68